tests/acceptance/features/apiGraph/enforcePasswordPublicLink.feature

```diff
@@ -0,0 +1,35 @@
+@api @env-config
+Feature: enforce password on public link
+  As a user
+  I want to enforce passwords on public links shared with upload, edit, or contribute permission
```

Comment:
```suggestion
  I want to enforce passwords on public links 
```

Reply:
not all public link gets password enforced. For now, password is enforced for links with upload,edit and contribute permissions. See https://doc.owncloud.com/ocis/next/deployment/services/s-list/sharing.html